Rubberduck.VBEEditor/Extensions/VbProjectExtensions.cs

```diff
@@ -1,3 +1,104 @@
-﻿namespace Rubberduck.VBEditor.Extensions
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Rubberduck.VBEditor.SafeComWrappers.Abstract;
+
+namespace Rubberduck.VBEditor.Extensions
 {
+    public static class VBProjectExtensions
```

Comment:
Agreed - *ideally* the VBE and VB6 objects should share base classes and only explicitly implement interface members that are strongly typed to an IVB* interface - for example, 2 `CodePane` members share the same 'CountOfVisibleLines', 'Selection', etc., and the `Equals` overrides could be shared by simply changing the parameter to an `ICodePane`.  Ultimately, the WinEvents stuff should live in it's own class, and simply be a member of whatever uses one - i.e. `private IWinEvents _eventController` or something like that.  The event stuff was a bit of WIP that got pulled in because I was an idiot and forgot to branch, then wasn't configured right to cherry-pick the inspection result form fix...